offchainservice/src/main/java/pl/edu/agh/blockchain/offchainservice/repository/EmailRepository.java

```diff
@@ -0,0 +1,15 @@
+package pl.edu.agh.blockchain.offchainservice.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import pl.edu.agh.blockchain.offchainservice.model.User;
+
+import java.util.List;
+
+@Repository
+public interface EmailRepository extends JpaRepository<User, String> {
+
+    List<User> findByEmail(String mail);
```

Comment:
Email is the primary key, so I think we need a method that returns just one user, not a list.

Reply:
JpaRepostitory allows to define methods and use it without implementation, but 
it returns a list by default, I can't change it